src/components/NoteDetailsButton/NoteDetailsButtonComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import IconButton from '@material-ui/core/IconButton';
+import EditIcon from '@material-ui/icons/Edit';
+
+import { Link } from 'react-router-dom';
+
+const FlexContainer = styled.div`
+  position: absolute;
+  width: 100vw;
+  z-index: 100;
```

Comment:
Why 100?

Reply:
It's fast styles and unfinished styling, I was told not to concentrate too much on styles, so I decided to leave it for others to style. I will fix it later when finished with redux and sagas. I think I used **z-index** because the notes tree rendered with react-graph-vis will render on top of it (I think because it's rendered with canvas and after all react components are rendered).

Generally using **z-index** is not good, but in our case we just need a quick hack/fix.

You can use values like 2,3,4, etc., but it's hard to keep track of what is where. You can use numbers like 1000, but you quickly can run out of number.

Counting by hundreds seems like a reasonable choice. In our case we just want our button to be rendere above all other UI and markup.

Reply:
aha I see, okay.